### PR TITLE
chore: bump SDK to v0.24.0

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

**Comment by Keith-CY** [comment]
CI failed

**Comment by classicalliu** [comment]
> CI failed

Fixed

File: packages/neuron-wallet/src/services/wallets.ts [hunk 1]
```diff
@@ -489,10 +489,15 @@ export default class WalletService {
   }
 
   public signWitness = (witness: string, privateKey: string, txHash: string): string => {
+    const witnessArg: CKBComponents.WitnessArgs = {
+      lock: witness,
```
Comment (ashchan) [inline packages/neuron-wallet/src/services/wallets.ts]: Are names for `lock` and `witness` still appropriate?

Comment (classicalliu) [reply packages/neuron-wallet/src/services/wallets.ts]: What's your point, I don't quite understand what you mean.

**Pull request merged**